css/style.css

```diff
@@ -1,7 +1,93 @@
+html {
+	box-sizing: border-box; /*padding и border включаются в общую ширину и высоту элемента*/
+}
+
+*, *::before, *::after {
+	box-sizing: inherit;
+}
+
+body {
+	max-width: 1920px; /*резиновая ширина*/
+	min-width: 375px;
+	height: 100vh; /*значение высоты опр-ся по viewport*/
+	background: url(../img/iswanto-arif-448003-unsplash.jpg) no-repeat center; /*фоновая картинка*/
+	background-size: cover; /*картинка полностью покрывает блок при любых размерах*/
+	color: #E8EEE4; /*цвет шрифта*/
+	font: normal 22px/26px MuseoSansCyrl; /*стиль шрифта*/
+}
+
+/*header styles start*/
+
+.head { 
+	max-width: 1607px; /*резиновая ширина*/
+	margin: 0 auto; /*выравнивание по центру*/
+	display: flex; 
+	flex-wrap: wrap; /*перенос пунктов*/
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Project-Bali/pullRequest?prid=5455742)